ociregistry/ociauth/auth.go

```diff
@@ -135,7 +135,7 @@ func (a *StdAuthorizer) DoRequest(req *http.Request, requiredScope Scope) (*http
 
 // doRequest performs the given request on the registry r.
 func (r *registry) doRequest(ctx context.Context, req *http.Request, requiredScope, wantScope Scope) (*http.Response, error) {
-	// TODO set up request body so that we can rewind it when retrying if necessary.
+	// TODO set up request body so that we can rewind it when retrying if necessary (http.NewRequest[WithContext] already does this for several types: https://pkg.go.dev/net/http#NewRequestWithContext).
```

Comment:
could you please make this comment two lines now? otherwise it's too long of a line